taxonomies/food/categories.txt

```diff
@@ -34265,6 +34265,7 @@ nl:Kruidnoten, Speculaasnoten
 <nl:Sinterklaasproducten
 nl:Taai taai
 
+<en:Meals
```

Comment:
```suggestion
```
 I don't think breaded products should be included in meals. For example, nuggets are not a meal.

Reply:
I do agree with you. I wanted to remove this lonely top and we have nothing at the moment.

What about introducing `meal components`?

Reply:
Almost every food can be part of a meal, I don't see the point. To me its not an issue that it has no parent. Is there a reason why you are trying to add many parents?